_data/ui/database-integration-settings/postgres.yml

```diff
@@ -11,6 +11,18 @@
 ## NOTE: 'defaults' is a variable assigned to this file: 
 ## /_data/ui/database-integration-settings/default-fields.yml
 
+
+field-names:
+  schema-names-tables: &schema-names-tables-name "Include PostgreSQL schema names in destination tables"
+
+descriptions:
+  schema-names-tables: &schema-names-tables-description |
+    Checking this setting will include schema names from the source database in the destination table name - for example: `<source_schema_name>__<table_name>`.
+
+          Stitch loads all selected replicated tables to a single schema, preserving only the table name. If two tables canonicalize to the same name - even if they're in different source databases or schemas - name collision errors can arise. Checking this setting can prevent these errors. 
```

Comment:
I think we should say "Checking this setting can prevent these issues." since it might not actually result in errors.

Reply:
Good point :+1: